package.json

```diff
@@ -20,10 +20,12 @@
   "dependencies": {
     "@stencil/core": "^0.7.10",
     "@stencil/router": "latest",
+    "@types/scroll-into-view": "^1.6.5",
     "bootstrap": "^4.0.0",
     "jquery": "^3.3.1",
     "popper": "^1.0.1",
-    "popper.js": "^1.14.1"
+    "popper.js": "^1.14.1",
+    "smoothscroll-polyfill": "^0.4.3"
```

Comment:
So we're included the polyfill dependency, but where do we load it in our project?

From the [browser support section](https://github.com/iamdustan/smoothscroll#browser-support), The scrolling behavior works natively (without the use of this polyfill) only on two browsers (chrome + firefox). But we need to ["use the polyfill"](https://github.com/iamdustan/smoothscroll#installation-and-use) to guarantee support on the remaining browsers.